OutSystems/outsystems-datagrid
# ROU-4407: Open link with middle button on action column

This PR is for allow using the middle mouse button to open link on the action columns

### What was done
* Added a validation to element type link, to apply different attributes based on externalURL to allows to redirect to an external URL on an action column, using a URL from data or a static URL. If the ExternalURL are being defined, the click event will not be triggered.

### Test Steps
**Test case 1**
1. Open sample page
2. Open each link with middle mouse button
3. Expected: Each link will open in a separated page

**Test Case 2**
1. Open sample page
2. On last column, click on link
3. Expected: The click event will be triggered and a feedback message will appear

### Checklist
